test/test_geo_utils.py

```diff
@@ -1,62 +1,18 @@
 """
 Test geo utils.
 """
-from qgis.core import QgsPointXY
-from qgis.core import QgsVectorLayer
 from ThreeDiToolbox.test.utilities import ensure_qgis_app_is_initialized
-from ThreeDiToolbox.utils.geo_utils import calculate_perpendicular_line
 from ThreeDiToolbox.utils.geo_utils import get_coord_transformation_instance
-from ThreeDiToolbox.utils.geo_utils import get_distance
-from ThreeDiToolbox.utils.geo_utils import get_epsg_code_from_layer
 
-import unittest
 
-
-class TestGeoUtils(unittest.TestCase):
-    def setUp(self):
-        ensure_qgis_app_is_initialized()
-
-    def test_it_can_get_epsg_code_from_layer(self):
-        crs = "EPSG:28992"
-        pnt_layer = QgsVectorLayer("Point?crs=" + crs, "temp_connected_pnt", "memory")
-        epsg_code = get_epsg_code_from_layer(pnt_layer)
-        self.assertEqual(epsg_code, 28992)
-
-    def test_it_can_get_coord_transformation_instance(self):
-        src_epsg, dest_epsg = 4326, 28992
-        inst = get_coord_transformation_instance(src_epsg, dest_epsg)
-        self.assertEqual(
-            str(inst.__class__), "<class 'qgis._core.QgsCoordinateTransform'>"
-        )
-        self.assertTrue(inst.destinationCrs().isValid())
-        self.assertTrue(inst.sourceCrs().isValid())
-        self.assertEqual(inst.destinationCrs().authid(), "EPSG:28992")
-        src_epsg, dest_epsg = 28992, 4326
-        inst_rev = get_coord_transformation_instance(src_epsg, dest_epsg)
-        self.assertEqual(inst_rev.destinationCrs().authid(), "EPSG:4326")
-
-    def test_it_can_get_distance_between_points(self):
-        pnt, pnt1 = QgsPointXY(0, 0), QgsPointXY(0, 10)
-        dist = get_distance(pnt, pnt1, 4326)
-        self.assertEqual(dist, 1105854.8332357334)
-        dist = get_distance(pnt, pnt1, 28992)
-        self.assertEqual(dist, 10.0)
-
-    def test_it_can_calculate_perpendicular_line(self):
-        line_coords = [0, 5, 0, 10]
-        expected = (-20.0, 5.0, 20.0, 5.0)
-        perpendicular_line = calculate_perpendicular_line(line_coords, 20)
-        self.assertEqual(expected, perpendicular_line)
-        perpendicular_line_left = calculate_perpendicular_line(line_coords, 20, "left")
-        expected_left = (0, 5, -20.0, 5.0)
-        self.assertEqual(expected_left, perpendicular_line_left)
-        expected_right = (0, 5, 20.0, 5.0)
-        perpendicular_line_right = calculate_perpendicular_line(
-            line_coords, 20, "right"
-        )
-        self.assertEqual(expected_right, perpendicular_line_right)
-
-        # should return None when the distance between the points <= 0
-        line_coords = [0, 0, 0, 0]
-        L = calculate_perpendicular_line(line_coords, 20)
-        self.assertIsNone(L)
+def test_it_can_get_coord_transformation_instance():
+    ensure_qgis_app_is_initialized()
+    src_epsg, dest_epsg = 4326, 28992
+    inst = get_coord_transformation_instance(src_epsg, dest_epsg)
+    assert (str(inst.__class__) == "<class 'qgis._core.QgsCoordinateTransform'>")
```

Comment:
Je kan ook `__name__` gebruiken. Maakt voor deze test niet uit, maar het is minder gevoelig voor eventuele refactorslagen aan qgis-kant (vooral omdat `_core` suggereert dat het intern spul is, raar genoeg). En het is korter.

```suggestion
    assert str(inst.__name__) == "QgsCoordinateTransform"
```

Wat ook vaak gebeurd: `assert isinstance(inst, QqsCoordinateTransform)`.